docs/operator-guides/configuring-airbyte-db.md

```diff
@@ -0,0 +1,81 @@
+# Configuring Airbyte Internal Database
+
+Airbyte uses different objects to store internal states and metadata. This data is stored and manipulated by the various Airbyte components, but you have the ability to manage how to deploy this database by using either:
+- By default, Airbyte spins-up its own Postgres database as part of one of th Docker service as described in its `docker-compose.yml` file: `airbyte/db`.
+- Dedicated custom Postgres instance.
+
+The various entities persisted in this internal database can be categorized as:
+
+1. Jobs Database: Data about executions of Airbyte Jobs and various runtime metadata.
+2. Temporal Database: Data about the internal orchestrator used by Airbyte, Temporal.io (Tasks, Workflow data, Events, and visibility data).
+3. Connection Configuration Database: Connectors, Sync Connections and various Airbyte configuration objects.
+
+Note that no actual data from the source (or destination) connectors ever transits or is retained in this internal database.
+
+If you need to interact with it, for example to make back-ups or perform some clean-up maintenances, you can also access to Export and Import functionalities of this database via API, or the UI (in the Admin page, in the Configuration Tab).
+
+## Connecting to some Postgres database
+
+Let's say we want to use a Postgres instance that is not automatically started by Airbyte, we can for example run the following command:
+```bash
+docker run --rm --name airbyte-postgres -e POSTGRES_PASSWORD=password -p 3000:5432 -d postgres
+```
+
+In order to configure Airbyte services with this new database, we need to edit the following environment variables declared in the `.env` file (used by the docker-compose command afterward):
+
+```bash
+DATABASE_USER=postgres
+DATABASE_PASSWORD=password
+DATABASE_HOST=host.docker.internal # refers to localhost of host
+DATABASE_PORT=3000
+DATABASE_DB=postgres
+```
+
+Additionally, you are free to redefine the JDBC URL constructed in the environment variable `DATABASE_URL` if you need to provide extra arguments to the Jdbc driver (for example, to handle SSL):
+
+```bash
+DATABASE_URL=jdbc:postgresql://${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DB}?ssl=true&sslmode=require
+```
+
+## Initializing the database
+
+{% hint style="info" %}
+This step is only required when you setup Airbyte with a custom database for the first time.
+{% endhint %}
+
+If you provide an empty database to Airbyte and start Airbyte up for the first time, the server and scheduler services won't be able to successfully start at this point as there is no data in the database yet.
```

Comment:
```suggestion
If you provide an empty database to Airbyte and start Airbyte up for the first time, the server and scheduler services won't be able to start because there is no data in the database yet.
```